### ISSUE TITLE
Gentoo ebuild failure

### ISSUE THREAD

**Issue by FireBurn** [yuzu-mirror/yuzu#9]
### Issue

The gentoo ebuild uses https://api.yuzu-emu.org/gamedb/ for it's compatibility list

It's on wayway back machine, is there any chance you could add it to a repo?

I'm not sure how it was actually generated

**Comment by FireBurn** [comment]
[gamedb.gz](https://github.com/yuzu-mirror/yuzu/files/14500247/gamedb.gz)

**Comment by FireBurn** [comment]
That's the download gziped

**Comment by Tatsh** [comment]
@FireBurn this was fixed last night. I just use an old copy from November. That's the latest on archive.org.

**Comment by FireBurn** [comment]
> @FireBurn this was fixed last night. I just use an old copy from November. That's the latest on archive.org.

In your ebuild, not mine :P I'll take a look at what you've done and incorporate it

**Comment by ascpixi** [comment]
https://github.com/yuzu-mirror/yuzu/blob/efe43c0f2882824e2cd127ae484c815f5ca9a0fd/CMakeLists.txt#L213-L218

Does the issue occur here? If so, I think you should be able to fix it by making `ENABLE_COMPATIBILITY_LIST_DOWNLOAD` false (or undefining it).

**Comment by Tatsh** [comment]
No. The error was in the ebuild itself. Yuzu requires the compatibility database or an empty one to build. The flag you mentioned is already used in the ebuild.

**Comment by ascpixi** [comment]
Ah, sorry, I'm not quite familiar with Gentoo and how it does things. I'm afraid I can't really help here, so if someone wants to make a PR fixing this, I'll be happy to merge it and, if needed, to create a repo hosting the database.

**Comment by koko1ooo** [comment]
The ebuild has been deleted from the guru repository, so this issue can be closed from my point of view. Apart from the fact that this issue seems to be a problem of the ebuild.

**Comment by FireBurn** [comment]
My ones in the FireBurn overlay and works fine now, but it would be good to get the file hosted somewhere and updated again

**Comment by ascpixi** [comment]
> My ones in the FireBurn overlay and works fine now, but it would be good to get the file hosted somewhere and updated again

I'd be happy to create a repository in this organization that would host these kinds of files, if there's a need for one.
